### DISCUSSION TITLE
Is there an easier way to just insert one character or if possible one word and exit insert mode.

### DISCUSSION THREAD

**Discussion by sanketwagh147** [VSCodeVim/Vim#7200] [Q&A] [answered]
Suppose I have the word 'login'.If the cursor is below **n** and I use  `r` it replaces it and exits insert.
Instead of that is there any subtle way to insert one character after **n** and exit insert so that it becomes 'login1'

Ps: love the extensions. Thank you for everyone who made this happen

**Comment by sanketwagh147** [comment, marked as answer]
This helps 
https://stackoverflow.com/questions/69736686/is-there-a-particular-key-to-enter-one-character-and-exit-insert-mode-in-vim-vsc

**Comment by jima246** [comment]
If the cursor is on 'n' and you want to append a '1', then type 'a 1 <esc>'
